tests/utils/component_data_wrapper.py

```diff
@@ -0,0 +1,148 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Wrappers for the ComponentData.
+
+In the unit tests we have to mock component data with only few parameters.
+Current ComponentData definition doesn't allow for it.
+One way to to achieve that is to create protobuf message and use it to
+create ComponentData. But this approach is complex and not resilient
+for the changes in the protobuf (api) definition.
+This module wraps ComponentData so that only part of it can be initialized.
+It should be used in unit tests only.
```

Comment:
I'd suggest a different phrasing/description for this part:


This module allows the easy construction of mock `ComponentData` instances to be used in unit tests.
Usually, all parameters for an instance need to be defined, this module helps by allowing users to only specify the parameters they are interested in for their tests. The rest will be filled with reasonable defaults.
This also abstracts away changes in the protobuf definition and minimizes the places that will need to be updated in such cases.

Reply:
Done, thanks a lot!